Services/DLNAService.m

```diff
@@ -788,12 +791,26 @@ -(NSDictionary*)getMetaDataDictionary:(NSString *)metaDataXML{
     if([mediaMetadataResponse objectForKey:@"dc:description"])
         [mediaMetaData setObject:[[mediaMetadataResponse objectForKey:@"dc:description"] objectForKey:@"text"] forKey:@"subtitle"];
     
-    if([mediaMetadataResponse objectForKey:@"upnp:albumArtURI"])
-        [mediaMetaData setObject:[[mediaMetadataResponse objectForKey:@"upnp:albumArtURI"] objectForKey:@"text"] forKey:@"iconURL"];
+    if([mediaMetadataResponse objectForKey:@"upnp:albumArtURI"]){
```

Comment:
This album art change is not related to playlists, so it should be in a different commit.